internal/db/changelog.go

```diff
@@ -1,38 +1,60 @@
 package db
 
 import (
-	"log"
+	"github.com/pkg/errors"
+
 	// Blank import required for mysql driver
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/jimmyjames85/bouncecm/internal/models"
 )
 
 // Changelog - Function to pull all rules from db
-func  (c *Client) Changelog() (*models.ChangelogTable, error) {
+func  (c *Client) GetAllChangelogEntries() (*[]models.ChangelogEntry, error) {
 	rules := []models.ChangelogEntry{}
 
 	rows, err := c.Conn.Query("SELECT * FROM changelog ")
 
 
 	if err != nil {
-		log.Println(err)
-		return nil, err;
+		return nil, errors.Wrap(err, "Query Error")
 	}
 
+	defer rows.Close()
+
 	for rows.Next() {
 		br := models.ChangelogEntry{}
 
 		err = rows.Scan(&br.ID,  &br.UserID,  &br.Comment,  &br.CreatedAt, &br.ResponseCode, &br.EnhancedCode, &br.Regex, &br.Priority, &br.Description, &br.BounceAction)
 		
 		if err != nil {
-			log.Println(err)
-			return nil, err;
+			return nil, errors.Wrap(err, "Changelog Row Scan")
 		}
 
 		rules = append(rules, br)
 	}
 
-	ChangelogTable := models.ChangelogTable{Rules: rules, NumRules: len(rules)}
-
-	return &ChangelogTable, nil
+	err = rows.Err()
+    if err != nil {
+        return nil, errors.Wrap(err, "Changelog Row Error")
+	}
+	
+	return &rules, nil
 }
+
+
+// func (c *Client) GetChangeLogEntry(id int) (*models.ChangelogEntry, error) {

```

Comment:
We should remove dead code. Eventually this branch will be merged to master, and we don't want to commit code that is commented.

Reply:
iImessed up branching when I was creating a new branch to work on the bounce rules. So this was suppose the with new pr i will remove it